client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/columns.tsx

```diff
@@ -117,20 +120,14 @@ export const useOutboundLineEditColumns = ({
     ],
   ];
   // If we have use VVM status, we need to show the VVM status column
-  if (
-    (prefs?.manageVvmStatusForStock || prefs?.sortByVvmStatusThenExpiry) &&
-    item?.isVaccine
-  ) {
+  if (prefs?.manageVvmStatusForStock && item?.isVaccine) {
```

Comment:
Removed sortByVVM status pref since the showing of VVM status should not be reliant on its sorting...